src/main/kotlin/com/exactpro/th2/common/schema/message/MessageRouterUtils.kt

```diff
@@ -103,8 +103,9 @@ fun MessageGroupBatch.toShortDebugString(): String = buildString {
     append(')')
 }
 
+private val subscribeArray = arrayOf(QueueAttribute.SUBSCRIBE.toString())
```

Comment:
```suggestion
private val SUBSCRIBE_DEFAULT_ATTRIBUTES = arrayOf(QueueAttribute.SUBSCRIBE.toString())
```
Also, please move to the `companion object` block